rest/changes_api_test.go

```diff
@@ -1183,6 +1335,9 @@ func TestChangesActiveOnlyWithLimit(t *testing.T) {
 	}
 
 	// Active only NO Limit, POST
+	testDb := it.ServerContext().Database("db")
+	testDb.FlushChannelCache()
```

Comment:
I think it's worth calling out why the `FlushChannelCache()` is being run since it's crucial to the test.  I guess it forces a view requery which surfaces the bug?

Reply:
I'll stick that in the test header